### ISSUE TITLE
Intelsense D455 vo error

### ISSUE THREAD

**Issue by jsreveal** [sair-lab/AirSLAM#173]
Hi there. You have mentioned in another issue that you tested with Realsense D455 without noticing major issues. I am testing now with RealSense D455, but I am experiencing errors in specially the angle of turning.

Which topics/streams did you use? I am using the two stereo infra1 and infra2, with the following calibration config. It is able to detect many stereo points, so I assume config is ok:
<img width="539" alt="image" src="https://github.com/user-attachments/assets/8b8dccf8-64d8-41a0-b0ec-10dd271d0ef5">

basically I recorded supermarket corridors that are of the same size, and perfectly aligned in parallel to each other, but the odometry result is the following:

<img width="265" alt="image" src="https://github.com/user-attachments/assets/2af0f966-6872-4bb9-aed8-5a8dbf54a1ce">

Thank you!

**Comment by xukuanHIT** [comment]
@jsreveal Hi, I think there is an error in either the image_width or the cx parameter you wrote. cx should roughly be half of the image_width.

**Comment by jsreveal** [comment]
oh man! thanks a lot. I was using the intrinsics from the 640x480 by mistake. Now this looks like something!
